config/prod/prow/config_knative.yaml

```diff
@@ -334,6 +334,11 @@ presubmits:
     args:
     - --run-test
     - ./test/e2e-upgrade-tests.sh
+  knative-sandbox/async-component:
+  - build-tests: false
```

Comment:
Could you explain why we don't need build-tests for this repo?

Reply:
I think we will need build tests, but I left this as false since they don't exist yet. Same for integration tests.

Reply:
build-tests will just run `go build -v ./...` by default, I think it shouldn't hurt to add now as long as the repo contains Go code?

Reply:
okay, thanks. That makes sense.

Reply:
Hmm, you probably forgot to run `./hack/generate-configs.sh`

Reply:
thanks! just ran & pushed the generated file.